core/src/main/java/org/kohsuke/stapler/export/PythonDataWriter.java

```diff
@@ -45,13 +45,13 @@ public void valueNull() throws IOException {
         data("None");
     }
 
-    @Override
+    /*@Override
```

Comment:
Please explain how it helps anyone to have a block of commented code left in the file?  Alternately, remove it if you are proposing to delete it.
```suggestion
    /*@Override
```